python/dgl/graphbolt/subgraph_sampler.py

```diff
@@ -193,7 +201,9 @@ def _seeds_preprocess(minibatch):
                 seeds_timestamp = torch.cat(
                     (minibatch.timestamp, neg_timestamp)
                 )
-                nodes_timestamp = [seeds_timestamp for _ in range(seeds.ndim)]
+                nodes_timestamp = [
+                    seeds_timestamp for _ in range(seeds.shape[1])
```

Comment:
why change this?

Reply:
This is a bug left in the past. This operation is used to make the number of timestamp same as the number of seeds. Previously, `seeds.ndim` = `seeds.shape[1]`, so we didn't find this bug. So, now we need to correct `seeds.ndim` to `seeds.shape[1]`.